src/routes/LandingPage/AboutUsSection/AboutUsSection.jsx

```diff
@@ -3,36 +3,42 @@ import './index.less';
 
 import { Typography } from 'antd';
 import placeholder from '../../../assets/about_us_placeholder.png';
+import { Router } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
This should just destructure one import
```suggestion
import { Link, Router } from 'react-router-dom';
```